### PR TITLE
refactor(up): bump fallback versions

### PR DESCRIPTION
**Pull request by evilrobot-01**
Bumps the fallback versions of the relay, parachains and chain specs.

### CODE REVIEW

**Comment by peterwht** [comment]
not sure if still good to review, but out of curiosity, any reason not to kickoff another release in our polkadot fork to use 2412?

**Comment by evilrobot-01** [comment]
> not sure if still good to review, but out of curiosity, any reason not to kickoff another release in our polkadot fork to use 2412?

These are only the fallback versions in case it cant connect to GH - e.g. rate limited. It should still check for a newer version if it exists. No reason not to kick off the stable2412 release, will give it a go.

File: crates/pop-parachains/src/up/mod.rs [hunk 1]
```diff
@@ -907,12 +906,12 @@ id = 1000
 chain = "asset-hub-paseo-local"
 "#
 			)?;
-			let system_parachain_version = "v1.12.0";
+			let system_parachain_version = "stable2407";
```
Comment (peterwht) [inline crates/pop-parachains/src/up/mod.rs:909]: Is there a reason this is `2407` instead of `2409?

**Review by peterwht — APPROVED** [review]
Left one comment. But I am going to approve as I am sure there is a reason for the 2407 and I don't want my timezone to block.

File: crates/pop-parachains/src/up/mod.rs (hunk 1 above)
Comment (evilrobot-01) [reply crates/pop-parachains/src/up/mod.rs:909]: Its just testing that a different version for a system parachain is used from the relay version.

**Review by AlexD10S — APPROVED** [review]
Looks good!

**Pull request merged**